<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EvilPostman</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_widget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="start_tab">
       <attribute name="title">
        <string>Start</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="2" column="0">
         <widget class="QLabel" name="start_tab_label_instruction">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;br/&gt;Modyfikacja pakietów przebiega w nasŧępujący sposób: &lt;br/&gt;A) Metoda szybka:&lt;br/&gt;B) Metoda zaawansowana: &lt;br/&gt;&lt;br/&gt;1) Przechodzimy dalej i włączamy przechwytywanie pakietów:&lt;br/&gt;2) Sposoby tworzeni a filtrów:&lt;/p&gt;&lt;p&gt;Metoda szybka: &lt;br/&gt;Klikamy prawym przyciskiem myszy na pakiecie i wybieramy utwórz szablon. Przechodzimy do wygenerowanego filtra i wybieramy wartości które mają zostać zmodyfikowane.&lt;br/&gt;Następnie akceptujemy zmiany i od tego momentu pakiety będą przechwytywane i modyfikowane.&lt;/p&gt;&lt;p&gt;Metoda zaawansowana: &lt;br/&gt;Klikamy na przycisk utwórz filtry&lt;br/&gt;Przechodzimy do wygenerowanego filtra i wybieramy wartości które mają zostać zmodyfikowane &lt;/p&gt;&lt;p&gt;Następnie akceptujemy zmiany i od tego momentu pakiety będą przechwytywane&lt;br/&gt;i modyfikowane.&lt;/p&gt;&lt;p&gt;4) Wysyłamy pakiet&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="start_tab_button_next">
          <property name="text">
           <string>Dalej</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="start_tab_label_title">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Tryb działania modyfikatora pakietów:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sniff_tab">
       <attribute name="title">
        <string>Przechwytywanie</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="1">
         <widget class="QPushButton" name="sniff_tab_button_create_filter_from_packet">
          <property name="text">
           <string>Utwórz filtr na podstawie pakietu</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="sniff_tab_button_next">
          <property name="text">
           <string>Dalej</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="sniff_tab_button_sniff">
          <property name="text">
           <string>Włącz przechwytywanie pakietów</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="sniff_tab_button_back">
          <property name="text">
           <string>Powrót</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTableWidget" name="sniff_tab_packets_list">
          <column>
           <property name="text">
            <string>Nr.</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Packet</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="filters_tab">
       <attribute name="title">
        <string>Filtry</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="4" column="2">
         <widget class="QPushButton" name="filters_tab_button_next">
          <property name="text">
           <string>Dalej</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="filters_tab_button_create_filter">
          <property name="text">
           <string>Utwórz filtr</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="filters_tab_button_back">
          <property name="text">
           <string>Powrót</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="filters_tab_list_of_packets">
          <column>
           <property name="text">
            <string>Nr.</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Filtr</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Aktywny</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modified_tab">
       <attribute name="title">
        <string>Lista zmodyfikwanych</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QPushButton" name="modified_tab_button_show">
          <property name="text">
           <string>Pokaż pakiet</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="modified_tab_back">
          <property name="text">
           <string>powrót</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="modified_tab_button_stop">
          <property name="text">
           <string>Zatrzymaj działanie</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="modified_tab_list_of_packets">
          <column>
           <property name="text">
            <string>Nr.</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Filtr</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Aktywny</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionETHERNET">
   <property name="text">
    <string>&amp;Filtry</string>
   </property>
  </action>
  <action name="actionWLAN">
   <property name="text">
    <string>&amp;Interfejsy</string>
   </property>
  </action>
  <action name="actionFiltry">
   <property name="text">
    <string>Filtry</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sniff_tab_button_sniff</sender>
   <signal>clicked()</signal>
   <receiver>sniff_tab_button_sniff</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>627</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
